config/ftbquests/quests/chapters/utilities.snbt

```diff
@@ -200,7 +200,7 @@
 		}
 		{
 			description: [
-				"The popular &edirewolf20's Building Gadgets&f have been thankfully ported to &dFabric&f and renamed under &bst'ructure tools&f. These function similarly to the &eBuilding Gadget&f versions and will be insanely helpful in building."

```

Comment:
this is the name of the mod. Don't ask me why but it is branded under `st'ructure tools`